document/ug/cfgprm.tex

```diff
@@ -0,0 +1,21 @@
+\begin{table}[H]
```

Comment:
the only file that can be added is custom.tex 

you cannot add other files as you cannot modify ug.tex

Reply:
same comment for regs.tex 

However, if your custom.tex includes these files it should be ok